src/test/kotlin/com/zopa/ktor/opentracing/UtilsTest.kt

```diff
@@ -0,0 +1,129 @@
+package com.zopa.ktor.opentracing
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Test
+
+class UtilsTest {
```

Comment:
Can we please keep the test classes as they are and add the tests for this in KtorOpenTracingTest . I agree they need to be split but would rather do in a separate PR.

Reply:
Sure 👍 I'll pull it out into a separate PR

Reply:
excellent , thanks ! :D